extensions/bear/src/menu-bar.tsx

```diff
@@ -0,0 +1,65 @@
+import {
+  Color,
+  Icon,
+  launchCommand,
+  LaunchType,
+  MenuBarExtra,
+  open,
+  openCommandPreferences,
+  showToast,
+  Toast,
+} from "@raycast/api";
+import { useEffect, useState } from "react";
+import { Note } from "./bear-db";
+import { useBearDb } from "./hooks";
+
+export default function MenuBar() {
+  const [db, error] = useBearDb();
+  const [notes, setNotes] = useState<Note[]>();
+
+  useEffect(() => {
+    if (db != null) {
+      setNotes(db.getNotes(""));
+    }
+  }, [db]);
+
+  if (error) {
+    showToast(Toast.Style.Failure, "Something went wrong", error.message);
+  }
+
+  return (
+    <MenuBarExtra isLoading={notes == undefined} icon={{ source: "bear-icon.png", tintColor: Color.PrimaryText }}>
+      {notes?.map((note) => (
+        <MenuBarExtra.Item
+          key={note.id}
+          title={note.title === "" ? "Untitled Note" : note.encrypted ? "🔒 " + note.title : note.title}
```

Comment:
Hey @thomaslombart,

I think it would be great to have the title be truncated here, to improve UX/UI. Perhaps truncating with an ellipsis would be sufficient after an X number of characters

Reply:
Good idea, I'll add support for that.